src/notation/internal/notationactioncontroller.cpp

```diff
@@ -1211,6 +1212,27 @@ void NotationActionController::addImage()
     interaction->addImageToItem(path, item);
 }
 
+void NotationActionController::replaceImage()
+{
+    auto interaction = currentNotationInteraction();
+    if (!interaction) {
+        return;
+    }
+
+    EngravingItem* item = contextItem(interaction);
+    if (!item->isImage()) {
+        return;
+    }
+
+    std::vector<std::string> filter = { trc("notation", "All Supported Files") + " (*.svg *.jpg *.jpeg *.png)",
```

Comment:
Here I see a code duplication problem. 
If someone decides to make a change to this code(adding another filter) and forgets to make a change to addImage, then we will get a different behavior - a bug.

I propose to move the common parts into separate method(s) and reuse them in addImage and replaceImage methods. 

Same for NotationInteraction::addImage and NotationInteraction::replaceImage